modules/nf-core/glimpse2/phase/main.nf

```diff
@@ -0,0 +1,69 @@
+process GLIMPSE2_PHASE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    beforeScript  """
+    if cat /proc/cpuinfo | grep avx2 -q
+    then
+        echo "Feature AVX2 present"
+    else
+        echo "Feature AVX2 not present on node"
+        exit 1
+    fi
+    """
```

Comment:
Yeah sure I just added it because otherwise the error message was obscure when running on an host that has not the feature available.
But what do you mean exactly by overwritten ?

Reply:
Yeah completely !
The aim is to test for the presence of the avx2 feature in the /proc/cpuinfo file.
You could use it differently by testing for another feature if you know how it is written in this file.
You therefore just need to overwrite avx2 in the grep command by whatever you need.